code/controllers/configuration.dm

```diff
@@ -76,10 +76,11 @@
 
 	var/server
 	var/banappeals
-	var/wikiurl = "http://www.tgstation13.org/wiki" // Default wiki link.
-	var/forumurl = "http://tgstation13.org/phpBB/index.php" //default forums
-	var/rulesurl = "http://www.tgstation13.org/wiki/Rules" // default rules
-	var/githuburl = "https://www.github.com/tgstation/-tg-station" //default github
+	var/wikiurl = "https://www.oraclestation.com/wiki/Main_Page" // Default wiki link.
+	var/wikibooksurl = "http://www.oraclestation.com" // Default wiki link.

```

Comment:
@FlattestGuitar Nope. BYOND doesn't support HTTPS and it needs to hit a different URL to start because of the API stuff. `wikiurl` is used for the button on the top right of the screen players can press, so I don't want to mess with it!